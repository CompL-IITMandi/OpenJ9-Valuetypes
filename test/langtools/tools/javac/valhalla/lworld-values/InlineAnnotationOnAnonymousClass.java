/*
 * @test /nodynamiccopyright/
 * @summary Check inlineness via __inline__ annotation
 * @bug 8222745
 * @compile/fail/ref=InlineAnnotationOnAnonymousClass.out -XDrawDiagnostics InlineAnnotationOnAnonymousClass.java
 */

class InlineAnnotationOnAnonymousClass {
    interface I {}
    @__inline__
    public static void main(String args []) {
        new @__inline__ I() {
        };
    }
}
