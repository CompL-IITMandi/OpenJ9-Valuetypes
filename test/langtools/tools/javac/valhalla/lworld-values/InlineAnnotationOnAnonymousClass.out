InlineAnnotationOnAnonymousClass.java:11:24: compiler.err.mod.not.allowed.here: inline
InlineAnnotationOnAnonymousClass.java:12:29: compiler.err.empty.value.not.yet
2 errors
